app/src/main/java/com/example/androidacademyhw/details/FragmentMoviesDetails.kt

```diff
@@ -53,8 +49,20 @@ class FragmentMoviesDetails : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        initAdapter()
+
         val movie = arguments?.getSerializable(Tags.KEY_MOVIE) as Movie
+        updateUI(movie)
+
+        binding.buttonBackImage.setOnClickListener { navigateBack() }
+        binding.buttonBackText.setOnClickListener { navigateBack() }
+    }
+
+    private fun navigateBack() {
+        requireActivity().supportFragmentManager.popBackStack()
+    }
 
+    private fun updateUI(movie: Movie) {
         with(binding) {
 
```

Comment:
remove empty line. and in general try to avoid extra empty lines, if u think that code is unreadable it is a sign that maybe u should split the method into small ones or rethink code style

Reply:
Sure. Got it